plenum/server/node.py

```diff
@@ -894,9 +896,6 @@ def on_view_change_start(self):
         self.monitor.reset()
         self.processStashedMsgsForView(self.viewNo)
 
-        for replica in self.replicas.values():
```

Comment:
In this PR, primary selection separated from view_changer, so that we can do all primaries changes at the end of a catchup.

Reply:
this functionality was separated in drop_primaries func. It still called during view_change, but in the end of catchup

Reply:
I think it needs to be called at the beginning of view change, not at the end of catchup.

Reply:
fixed